findbugs-exclude.xml

```diff
@@ -0,0 +1,5 @@
+<FindBugsFilter>
+     <Match>
+       <Class name="~.+\.Messages" />
```

Comment:
This is the error I was (and am) getting when running the findbugs check on this same PR (removing the `findbugs-maven-plugin` configuration override). Basically an error in the generated source.

```
[INFO] >>> findbugs-maven-plugin:3.0.3:check (findbugs) > :findbugs @ ant >>>
[INFO]
[INFO] --- findbugs-maven-plugin:3.0.3:findbugs (findbugs) @ ant ---
[INFO] Fork Value is true
     [java] Warnings generated: 1
[INFO] Done FindBugs Analysis....
[INFO]
[INFO] <<< findbugs-maven-plugin:3.0.3:check (findbugs) < :findbugs @ ant <<<
[INFO]
[INFO] --- findbugs-maven-plugin:3.0.3:check (findbugs) @ ant ---
[INFO] BugInstance size is 1
[INFO] Error size is 0
[INFO] Total bugs: 1
[INFO] The method name hudson.tasks._ant.Messages.InstallFromApache() doesn't start with a lower case letter [hudson.tasks._ant.Messages] At Messages.java:[line 153] NM_METHOD_NAMING_CONVENTION
[INFO]


To see bug detail using the Findbugs GUI, use the following command "mvn findbugs:gui"
```

Anyway, thanks for the information. I will investigate this in a bit. :)